pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -137,7 +138,9 @@ func (admitter *VMsAdmitter) Admit(ctx context.Context, ar *admissionv1.Admissio
 			return webhookutils.ToAdmissionResponse(causes)
 		}
 	}
-	causes = ValidateVirtualMachineSpec(k8sfield.NewPath("spec"), &vmCopy.Spec, admitter.ClusterConfig, accountName)
+
+	_, isKubeVirtServiceAccount := admitter.KubeVirtServiceAccounts[ar.Request.UserInfo.Username]
```

Comment:
nit: WDYT about something like
```go
func isArOfKubevirtServiceAccount(ar *AdmissionReview, KubeVirtServiceAccounts map[string]struct{}) bool {}
```

Although I'm not sure it's more elegant than the current code.. WDYT?

Reply:
TBH, I don't see the benefit of adding this function at the moment.
The admitters will still have to contain the KubeVirt ServiceAccount map / set, and the complex validation logic mostly care if the change was made by KubeVirt or not.